ompi/mca/coll/tuned/coll_tuned_decision_fixed.c

```diff
@@ -404,10 +404,11 @@ int ompi_coll_tuned_alltoall_intra_dec_fixed(const void *sbuf, size_t scount,
         }
     }
 
+    int faninout = 2;
     return ompi_coll_tuned_alltoall_intra_do_this (sbuf, scount, sdtype,
                                                    rbuf, rcount, rdtype,
                                                    comm, module,
-                                                   alg, 0, 0, ompi_coll_tuned_alltoall_max_requests);
+                                                   alg, faninout, 0, ompi_coll_tuned_alltoall_max_requests);
```

Comment:
overload and reuse of the `fanout` mca parameter in this new implementation could be a source of confusion if it differs with the original purpose/meaning of the parameter. I suggest introducing a new mca parameter called `radix`

Reply:
@jiaxiyan Can you simply pass `2` to the function without using `faninout`?